wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -195,11 +192,16 @@ def _update_run_states(self) -> None:
                 ]:
                     run.state = SimpleRunState.ALIVE
             except Exception as e:
```

Comment:
If this happens relatively frequently you could try to catch the specific Exception here, especially if its a network/comm thing that happens non-deterministically.

Reply:
I think the only time this happens is when the run doesn't exist yet right? Still we should attempt to be as narrow as possible with our catch exceptions.